# HG changeset patch
# User Jean-Francois Dockes <jfd@recoll.org>
# Date 1381170223 -7200
# Node ID 79dcd7fa03797b6b51720025b3c3276395523c38
# Parent  7ea9fe8dc5be75557130245d7b24526a25376eb4
replace deprecated qt action::activated() with action::triggered()

diff --git a/src/qtgui/rclmain_w.cpp b/src/qtgui/rclmain_w.cpp
--- qtgui/rclmain_w.cpp
+++ qtgui/rclmain_w.cpp
@@ -233,51 +233,51 @@
 	    this, SLOT(setStemLang(QAction*)));
     connect(preferencesMenu, SIGNAL(aboutToShow()),
 	    this, SLOT(adjustPrefsMenu()));
-    connect(fileExitAction, SIGNAL(activated() ), 
+    connect(fileExitAction, SIGNAL(triggered() ), 
 	    this, SLOT(fileExit() ) );
-    connect(fileToggleIndexingAction, SIGNAL(activated()), 
+    connect(fileToggleIndexingAction, SIGNAL(triggered()), 
 	    this, SLOT(toggleIndexing()));
-    connect(fileRebuildIndexAction, SIGNAL(activated()), 
+    connect(fileRebuildIndexAction, SIGNAL(triggered()), 
 	    this, SLOT(rebuildIndex()));
-    connect(fileEraseDocHistoryAction, SIGNAL(activated()), 
+    connect(fileEraseDocHistoryAction, SIGNAL(triggered()), 
 	    this, SLOT(eraseDocHistory()));
-    connect(fileEraseSearchHistoryAction, SIGNAL(activated()), 
+    connect(fileEraseSearchHistoryAction, SIGNAL(triggered()), 
 	    this, SLOT(eraseSearchHistory()));
-    connect(helpAbout_RecollAction, SIGNAL(activated()), 
+    connect(helpAbout_RecollAction, SIGNAL(triggered()), 
 	    this, SLOT(showAboutDialog()));
-    connect(showMissingHelpers_Action, SIGNAL(activated()), 
+    connect(showMissingHelpers_Action, SIGNAL(triggered()), 
 	    this, SLOT(showMissingHelpers()));
-    connect(showActiveTypes_Action, SIGNAL(activated()), 
+    connect(showActiveTypes_Action, SIGNAL(triggered()), 
 	    this, SLOT(showActiveTypes()));
-    connect(userManualAction, SIGNAL(activated()), 
+    connect(userManualAction, SIGNAL(triggered()), 
 	    this, SLOT(startManual()));
-    connect(toolsDoc_HistoryAction, SIGNAL(activated()), 
+    connect(toolsDoc_HistoryAction, SIGNAL(triggered()), 
 	    this, SLOT(showDocHistory()));
-    connect(toolsAdvanced_SearchAction, SIGNAL(activated()), 
+    connect(toolsAdvanced_SearchAction, SIGNAL(triggered()), 
 	    this, SLOT(showAdvSearchDialog()));
-    connect(toolsSpellAction, SIGNAL(activated()), 
+    connect(toolsSpellAction, SIGNAL(triggered()), 
 	    this, SLOT(showSpellDialog()));
-    connect(indexConfigAction, SIGNAL(activated()), 
+    connect(indexConfigAction, SIGNAL(triggered()), 
 	    this, SLOT(showIndexConfig()));
-    connect(indexScheduleAction, SIGNAL(activated()), 
+    connect(indexScheduleAction, SIGNAL(triggered()), 
 	    this, SLOT(showIndexSched()));
-    connect(queryPrefsAction, SIGNAL(activated()), 
+    connect(queryPrefsAction, SIGNAL(triggered()), 
 	    this, SLOT(showUIPrefs()));
-    connect(extIdxAction, SIGNAL(activated()), 
+    connect(extIdxAction, SIGNAL(triggered()), 
 	    this, SLOT(showExtIdxDialog()));
 
     if (prefs.catgToolBar && catgCMB)
 	connect(catgCMB, SIGNAL(activated(int)), 
 		this, SLOT(catgFilter(int)));
-    connect(toggleFullScreenAction, SIGNAL(activated()), 
+    connect(toggleFullScreenAction, SIGNAL(triggered()), 
             this, SLOT(toggleFullScreen()));
-    connect(actionShowQueryDetails, SIGNAL(activated()),
+    connect(actionShowQueryDetails, SIGNAL(triggered()),
 	    reslist, SLOT(showQueryDetails()));
     connect(periodictimer, SIGNAL(timeout()), 
 	    this, SLOT(periodic100()));
 
     restable->setRclMain(this, true);
-    connect(actionSaveResultsAsCSV, SIGNAL(activated()), 
+    connect(actionSaveResultsAsCSV, SIGNAL(triggered()), 
 	    restable, SLOT(saveAsCSV()));
     connect(this, SIGNAL(docSourceChanged(RefCntr<DocSequence>)),
 	    restable, SLOT(setDocSource(RefCntr<DocSequence>)));
@@ -309,11 +309,11 @@
     reslist->setRclMain(this, true);
     connect(this, SIGNAL(docSourceChanged(RefCntr<DocSequence>)),
 	    reslist, SLOT(setDocSource(RefCntr<DocSequence>)));
-    connect(firstPageAction, SIGNAL(activated()), 
+    connect(firstPageAction, SIGNAL(triggered()), 
 	    reslist, SLOT(resultPageFirst()));
-    connect(prevPageAction, SIGNAL(activated()), 
+    connect(prevPageAction, SIGNAL(triggered()), 
 	    reslist, SLOT(resPageUpOrBack()));
-    connect(nextPageAction, SIGNAL(activated()),
+    connect(nextPageAction, SIGNAL(triggered()),
 	    reslist, SLOT(resPageDownOrNext()));
     connect(this, SIGNAL(searchReset()), 
 	    reslist, SLOT(resetList()));
